test/crypten/test_utils.py

```diff
@@ -54,6 +55,29 @@ def forward(self, x):
     assert th.all(expected_out == out)
 
 
+def test_pack_typerror_crypten_model():
+    dummy_input = th.rand(1, 28 * 28)
```

Comment:
Can you add a line to say what this test tests?

Reply:
Sure